components/chat/chatBar/search/catalog.tsx

```diff
@@ -0,0 +1,592 @@
+      url: 'sys.exec',
+      tags: ['system', 'execute', 'exec'],
+      icon: <GoTerminal className="text-lg" />,
+    },
+    {
+      name: 'Find',
+      description:
+        'Traverse a directory looking for files that match a pattern in the style of the unix find command.',
+      url: 'sys.find',
+      tags: ['system', 'find'],
+      icon: <BsFiles className="text-lg" />,
+    },
+    {
+      name: 'Get Environment Variable',
+      description: 'Gets the value of an OS environment variable.',
+      url: 'sys.getenv',
+      tags: ['system', 'getenv', 'environment', 'env'],
+      icon: <LuServer className="text-lg" />,
+    },
+    {
+      name: 'HTML to Text',
+      description:
+        'Download the contents of a http or https URL returning the content as rendered text converted from HTML.',
+      url: 'sys.http.html2text',
+      tags: ['system', 'http', 'html', 'text'],
+      icon: <FaCode className="text-lg" />,
+    },
+    {
+      name: 'HTTP GET',
+      description: 'Download the contents of a http or https URL.',
+      url: 'sys.http.get',
+      tags: ['system', 'http', 'get'],
+      icon: <GoGlobe className="text-lg" />,
+    },
+    {
+      name: 'HTTP POST',
+      description:
+        'Write contents to a http or https URL using the POST method.',
+      url: 'sys.http.post',
+      tags: ['system', 'http', 'post'],
+      icon: <GoGlobe className="text-lg" />,
+    },
+    {
+      name: 'List Directory',
+      description: 'Lists the contents of a directory.',
+      url: 'sys.ls',
+      tags: ['system', 'ls', 'list', 'directory'],
+      icon: <BsFolder className="text-lg" />,
+    },
+    {
+      name: 'Prompt',
+      description: 'Prompts the user for input.',
+      url: 'sys.prompt',
+      tags: ['system', 'prompt', 'input'],
+      icon: <GoQuestion className="text-lg" />,
+    },
+    {
+      name: 'Read File',
+      description: 'Reads the contents of a file.',
+      url: 'sys.read',
+      tags: ['system', 'read', 'file'],
+      icon: <FaGlasses className="text-lg" />,
+    },
+    {
+      name: 'Remove File',
+      description: 'Removes the specified file.',
+      url: 'sys.remove',
+      tags: ['system', 'remove', 'file'],
+      icon: <MdDeleteForever className="text-lg" />,
+    },
+    {
+      name: 'File Stat',
+      description: 'Gets size, modfied time, and mode of the specified file.',
+      url: 'sys.stat',
+      tags: ['system', 'stat', 'file'],
+      icon: <BsSearch className="text-lg" />,
+    },
+    {
+      name: 'Current Time',
+      description: 'Returns the current date and time in RFC3339 format.',
+      url: 'sys.time.now',
+      tags: ['system', 'time', 'now'],
+      icon: <BsClock className="text-lg" />,
+    },
+    {
+      name: 'Write File',
+      description: 'Write the contents to a file.',
+      url: 'sys.write',
+      tags: ['system', 'write', 'file'],
+      icon: <GoPencil className="text-lg" />,
+    },
+  ],
+  'From URL': [],
```

Comment:
Make sure empty sections don't show up in the listbox.

Reply:
removed